.github/workflows/test_pytorch_wheels.yml

```
@@ -30,8 +60,31 @@ jobs:
     name: Test PyTorch Wheels | ${{ inputs.amdgpu_families }}
     runs-on: ${{ inputs.test_runs_on }}
 
-    env:
-      PACKAGE_DIST_DIR: /tmp/wheels
     steps:
       - name: Checkout
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Set up Python
+        uses: actions/setup-python@42375524e23c412d93fb67b49958b491fce71c38 # v5.4.0
+        with:
+          python-version: '3.12'
```

If you want to make the Python version an input parameter you need to make sure to respect this here:
```suggestion
          python-version: ${{ inputs.python_version }}
```